### ISSUE TITLE
Example.scala fails to read files from Camel file component

### ISSUE THREAD

**Issue by searler** [krasserm/streamz#37]
The 
```
 val fileEndpointUri: String =
    "file:input?charset=utf-8"
```
must be changed to
```
 val fileEndpointUri: String =
    "file:input?noop=true"
```

otherwise the file is moved before it can be read.

**Comment by searler** [comment]
Obviously most applications cannot be restricted to "noop=true"

**Comment by krasserm** [comment]
@searler thanks for reporting, that was a bug which is now fixed in master. Can you please give it a try? When working for you, I'll make a 0.8.1 release in the next days.

**Comment by krasserm** [comment]
If message conversion fails, the corresponding exception is also set on the inbound CamelExchange (akka-stream and FS2)

**Comment by krasserm** [comment]
> that was a bug

I mean not the missing `noop=true` was the bug but the fact that it didn't work otherwise 😃

**Comment by searler** [comment]
Works now.

Thanks for the quick response

**Comment by krasserm** [comment]
You're welcome. I just released 0.8.1 with this bug fix.
